docs/photoniq/event-delivery/get-started-event-delivery.md

```diff
@@ -9,42 +9,40 @@ However, some people like to be more hands-on with their management. This page p
 
 A stream is a named channel for sending messages. Each stream is backed by a distributed append-only log and can be local (at one edge location only) or global (across all edge locations in the [fabric](../../geofabrics/)).
 
-For more information, refer to [Streams](../../streams/).
-
-### Collection Stream
+1. (Optional) [Create a fabric](../../geofabrics/create-geofabric.md) for your collection. This allows you to decide exactly where your data lives. If you skip this step, then the collection will probably be in the system fabric.
+2. Create a collection that you will send streaming data to. Make sure you enable streams when you create it. Instructions for each are below:
+   - [Create a Document Collection](../../collections/documents/create-document-store.md)
+   - [Create a Key-Value Collection](../../collections/keyvalue/create-key-value-store.md)
 
-Macrometa GDN has several types of collections, all of which can have the data to a stream as records are added.
+   ![New Collection Stream](/img/photoniq/event-delivery/new-collection-stream.png)
 
-For more information, refer to:
+3. [View collection settings](../../collections/view-collection-settings.md) to access the following information:
+   - **Resource URL** - The endpoint to send data to your collection. All messages sent here are both stored and transmitted to the stream.
+   - **Stream Name** - Parameter for Event Delivery endpoints.
+   - **Fabric Name** - Parameter for Event Delivery endpoints. In the screenshot below, the fabric name is **_system**.
 
-- [Collections](../../collections/)
-- [Collection Settings](../../collections/view-collection-settings#view-collection-settings) - View and modify the Collection Stream settings.
+   ![View Collection Settings](/img/photoniq/event-delivery/view-collection-settings.png)
 
 ## Set Up Event Delivery
 
 Once your stream is set up in the GDN, you can subscribe to it with Event Delivery and manage it using the [Event Delivery API](https://www.macrometa.com/docs/apiEds#/).
 
-1. [Subscribe to the stream](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get). This includes adding any [Event Delivery Queries](event-delivery-queries.md) to filter events.
-2. [Publish events](https://www.macrometa.com/docs/apiEds#/paths/api-es-v1-fabric-fabricName--stream--streamName--publish/post). You can use this API endpoint or use one of the options for publishing events to the GDN stream.
+1. [Subscribe to the stream](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get). This includes adding any [Event Delivery queries](event-delivery-queries.md) to filter events.
+2. [Publish events](https://www.macrometa.com/docs/apiEds#/paths/api-es-v1-fabric-fabricName--stream--streamName--publish/post). You can use this API endpoint or use one of the options for publishing events to the GDN stream. Events published in this matter are not stored in the collection.
```

Comment:
we could also mention how a GDN stream can be created like how its is mentioned for collection above.

Reply:
I'm not sure what you mean by that.

If you mean the GDN Streams feature, Durga wanted this topic simplified to just collection streams, so I took out the content talking about GDN streams.